### ISSUE TITLE
how to map out the cri log?

### ISSUE THREAD

**Issue by fyxemmmm** [fluent/fluent-bit#4032]
[PARSER]
        Name cri
        Format regex
        Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$
        Time_Key    time
        Time_Format %Y-%m-%dT%H:%M:%S.%L%z
        Decode_Field_As    json        message

when I use regex, 
I want to map out the message field, but it does not work, how can I do for that?

**Comment by ciastooo** [comment]
@fyxemmmm do you use this `cri` parser in your `[INPUT]`? An example of this could be:
```
    [INPUT]
        Name              tail
        Tag               kube.*
        Path              /var/log/containers/*.log
        Parser            cri
        DB                /var/log/flb_kube.db
        Mem_Buf_Limit     5MB
        Skip_Long_Lines   On
        Refresh_Interval  10
```

**Comment by fyxemmmm** [comment]
> @fyxemmmm do you use this `cri` parser in your `[INPUT]`? An example of this could be:
> 
> ```
>     [INPUT]
>         Name              tail
>         Tag               kube.*
>         Path              /var/log/containers/*.log
>         Parser            cri
>         DB                /var/log/flb_kube.db
>         Mem_Buf_Limit     5MB
>         Skip_Long_Lines   On
>         Refresh_Interval  10
> ```

yes, but  I want to map the regular parsed message data again, the docker parser can do it, but he regular exp can not do it.
